examples/knative/go/kn-go-echo/function.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-go-echo
+  labels:
+    app: veba-ui
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: projects.registry.vmware.com/veba/kn-go-echo:1.0
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: kn-go-echo-trigger
+  labels:
+    app: veba-ui
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: com.vmware.event.router/event
```

Comment:
Do we need that filter? (see comment below)

Reply:
I can get rid of the filter in favor of modifying the cloud event in a way that will avoid an infinite event loop.